### ISSUE TITLE
Error creating CylinderZeppinBall model

### ISSUE THREAD

**Issue by ErickHernandezGutierrez** [daducci/AMICO#92]
I am getting the following error in AMICO when I run COMMIT with CylinderZeppelinBall:

TypeError                                 Traceback (most recent call last)
~/.local/lib/python3.6/site-packages/numpy/core/function_base.py in linspace(start, stop, num, endpoint, retstep, dtype, axis)
```
    116     try:
--> 117         num = operator.index(num)
    118     except TypeError:
```
TypeError: 'float' object cannot be interpreted as an integer

During handling of the above exception, another exception occurred:
TypeError                                 Traceback (most recent call last)
/mnt/d/Documents/DiceLab/Phantom/generate-lut.py in <module>
```
     23 mit.load_data( 'data.nii.gz', 'dwi.scheme' )
     24
---> 25 mit.set_model( 'CylinderZeppelinBall' )
     26 d_par = 1.7E-3
     27 Ra     = np.array([0.1, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5]) * 1E-6  # IC radius in micro-
meters
commit/core.pyx in commit.core.Evaluation.set_model()
~/.local/lib/python3.6/site-packages/amico/models.py in __init__(self)
    312
    313         self.d_par  = 0.6E-3                                                         # Parallel diffusivity
 [mm^2/s]
--> 314         self.Rs     = np.concatenate( ([0.01],np.linspace(0.5,8.0,20.0)) ) * 1E-6    # Radii of the axons [
meters]
    315         self.ICVFs  = np.arange(0.3,0.9,0.1)                                         # Intra-cellular volum
e fraction(s) [0..1]
    316         self.d_ISOs = np.array( [ 2.0E-3 ] )                                         # Isotropic diffusivit
ie(s) [mm^2/s]
<__array_function__ internals> in linspace(*args, **kwargs)
~/.local/lib/python3.6/site-packages/numpy/core/function_base.py in linspace(start, stop, num, endpoint, retstep, dtype, axis)
    119         raise TypeError(
    120             "object of type {} cannot be safely interpreted as an integer."
```

I think the problem is that we have to change the line
`self.Rs     = np.concatenate( ([0.01],np.linspace(0.5,8.0,20.0)) ) * 1E-6`
by the line
`self.Rs     = np.concatenate( ([0.01],np.linspace(0.5,8.0,20)) ) * 1E-6`
to avoid the error as linspace() needs an integer number instead of a float number to specify the number of points we want in the discretization of the interval [0.5, 8.0].

**Comment by daducci** [comment]
Good catch! Indeed, this is the only part of the code where the `num` parameter is given as a float, probably a typo. Can you make a PR and fix this?

**Comment by daducci** [comment]
Fixed by #93
